src/components/BrowserSnifferService.js

```diff
@@ -28,6 +28,12 @@
       var webkit = !msie && /WebKit/.test(ua);
       var opera = !msie && /(OPiOS|OPR)\//.test(ua);
       var chrome = !msie && !opera && /(CriOS|Chrome)\//.test(ua);
+      if (chrome) {
+        var m = ua.match(/Chrome\/(\d+)\./);
```

Comment:
you mean adapt https://github.com/geoadmin/mf-geoadmin3/blob/master/test/specs/BrowserSnifferService.spec.js
accordingly ?